src/kontrol/kdist/cheatcodes.md

```diff
@@ -435,7 +435,28 @@ This rule returns a symbolic boolean value being either 0 (false) or 1 (true).
          <k> #call_foundry SELECTOR _ => . ... </k>
          <output> _ => #buf(32, ?WORD) </output>
       requires SELECTOR ==Int selector ( "freshBool()" )
-       ensures #rangeBool(?WORD)
+       ensures #rangeBool(?WORD2)
```

Comment:
I am not sure what happened here, `?WORD2` is not bound in the rule above. This should be `?WORD`.

Reply:
Fixed.